.github/workflows/workflow.yml

```diff
@@ -0,0 +1,56 @@
+name: Main workflow
+
+on:
+  push:
+    paths-ignore:
+      - '**.md'
+      - '.all-contributorsrc'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - '.all-contributorsrc'
+  schedule:
+    - cron: "0 0 * * 5"
+
+jobs:
+  plugin_test:
+    strategy:
+      matrix:
+        os: [macOS-latest, ubuntu-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: asdf_plugin_test
+        uses: asdf-vm/actions/plugin-test@v1.0.0
+        with:
+          command: lfe --version
```

Comment:
`lfe --version` -> `lfe -version`, but this give the information about the erlang vm. It's not ideal, but at least it pass through `lfe`.